fake-koji/src/main/java/org/fakekoji/api/http/rest/OToolService.java

```diff
@@ -57,7 +58,7 @@
     private static final String JDK_TEST_PROJECTS = "/jdkTestProjects";
     private static final String JDK_TEST_PROJECT = JDK_TEST_PROJECTS + CONFIG_ID;
     private static final String GET = "get";
-
+    private static final String BUMP = "bump";
```

Comment:
the bump must be accessible via  misc/bump MUST
thus the help is missing in misc help

Reply:
will do

Reply:
tyvm!